python/perf-kernels/flash-attention.py

```diff
@@ -202,6 +212,24 @@ def _attn_fwd_inner(
             # our optimization to use 2^x instead of e^x results in an additional
             # scale factor of log2(e) which we must also multiply the bias with.
             qk += (bias * 1.44269504089)
+           
+        if alibi_slopes_ptr is not None:
+            alibi_slopes_block = load_fn(alibi_slopes_ptr, False, MASK_STEPS and (n_extra_tokens != 0), "zero")
```

Comment:
I don't think it is possible to go out of bounds for this. So you can use False, False for the bounds check args, or provide no mask in the tl.load (depending on if you take my suggestion to not use a block pointer for this or not).

Reply:
I am using just a tl.load at the start of the kernel.